docs/writing-rules/rule-syntax.md

```diff
@@ -466,7 +466,7 @@ rules:
       discovered-by: Ikwa L'equale
 ```
 
-The metadata are also displayed in Semgrep’s output if you’re running it with `--json`.
+The metadata are also displayed in Semgrep output if you’re running it with `--json`.
```

Comment:
```suggestion
The metadata are also displayed in Semgrep output if you’re running it with `--json`.
```
"Semgrep output" sounds a little awkward, I think, particularly because Semgrep is a proper noun. I'll leave it to you if you want to fix it.

Reply:
```suggestion
The metadata are also displayed in the output of Semgrep if you’re running it with `--json`.
```